### PR TITLE
feat: store tweets and add status page

### PR DESCRIPTION
**Pull request by eliecharra**
For #25 

image

### CODE REVIEW

File: internal/domain/status/handler_test.go [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+	fakeTweet := &models.Tweet{
+		ID:        "1234",
+		Text:      "This is a fake tweet content",
+		Published: fakeDate,
+		Sensitive: false,
+	}
+
+	cases := []tests.HTTPTestCase{
+		{
+			Name: "missing tweet id",
```
Comment (smaftoul) [inline internal/domain/status/handler_test.go:51]: Not sure I understand how tests works here, but isn't this supposed to be missing tweet username ?

File: internal/domain/status/handler_test.go [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+		},
+		{
+			Name: "missing tweet id",
+			RequestOptions: []tests.RequestOption{
+				tests.RequestParams{Params: map[string]string{"id": fakeTweet.ID}},
+			},
+			StatusCode: http.StatusBadRequest,
+		},
+		{
+			Name: "twitter user no found",
```
Comment (smaftoul) [inline internal/domain/status/handler_test.go:65]: *not found ?

File: internal/domain/status/views/views.go [hunk 3]
```diff
@@ -0,0 +1,6 @@
+package views
+
+import "embed"
+
+//go:embed *
+var Views embed.FS
```
Comment (smaftoul) [inline internal/domain/status/views/views.go:6]: do we end up with this file in the FS ? do we care ? ;)

File: internal/router/routes/routes.go [hunk 4]
```diff
@@ -7,6 +7,6 @@ const (
 	UserFollowingRoute string = "user_following"
 	UserFollowersRoute string = "user_followers"
 	UserOutbox         string = "user_outbox"
-	UserStatuses       string = "user_statuses"
```
Comment (smaftoul) [inline internal/router/routes/routes.go:10]: thanks ;)

**Review by smaftoul — APPROVED** [review]

File: internal/domain/status/handler_test.go (hunk 1 above)
Comment (eliecharra) [reply internal/domain/status/handler_test.go:51]: Good catch thanks !

File: internal/domain/status/views/status.html [hunk 5]
```diff
@@ -0,0 +1,47 @@
+<html lang="en" class="h-100">
```
[the remark this answers is not in the document]
Comment (smaftoul) [reply internal/domain/status/views/status.html:1]: Should we then remove the attribute ?

**Review by smaftoul — APPROVED** [review]

File: internal/domain/status/views/status.html (hunk 5 above)
[the remark this answers is not in the document]
Comment (eliecharra) [reply internal/domain/status/views/status.html]: Yes !

**Pull request merged**